cg/constants/constants.py

```diff
@@ -258,6 +258,13 @@ class MicrosaltAppTags(StrEnum):
     PREP_CATEGORY: str = "mic"
 
 
+MICROBIAL_APP_TAGS = [
+    MicrosaltAppTags.MWRNXTR003,
+    MicrosaltAppTags.MWXNXTR003,
+    MicrosaltAppTags.VWGNXTR001,
+]
```

Comment:
With my suggestion above this is not required

```suggestion
```

Reply:
still required as microbial-fastq is not a delivery option so it will not be available to the factory